tests/formats/cssr.cpp

```diff
@@ -90,6 +90,10 @@ R"( REFERENCE STRUCTURE = 00000   A,B,C =  10.000  10.000  12.000
     frame.set_cell(UnitCell(10, 10, 12));
     auto t = Trajectory(tmpfile, 'w');
     t.write(frame);
+
+    // Format only supports writing one frame
+    CHECK_THROWS(t.write(frame));
```

Comment:
Maybe use `CHECK_THROWS_WITH` instead of `CHECK_THROW` to ensure the expected exception is being thrown?

Reply:
Good idea.